### PR TITLE
Add users API and updated Page Data

### PR DESCRIPTION
**Pull request by protsack-stephan**
- Add `PageData` method to get a single page data instead of many.
- Add `Users` method to query multiple users by id at once
- Add `User` method to query single user by id

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -272,3 +295,70 @@ func (cl *Client) Namespaces(ctx context.Context) ([]Namespace, error) {
 
 	return ns, nil
 }
+
+// Users get list of users by id.
+func (cl *Client) Users(ctx context.Context, ids ...int) (map[int]User, error) {
+	ususerids := []string{}
+
+	for _, id := range ids {
+		ususerids = append(ususerids, strconv.Itoa(id))
```
Comment (Drynok) [inline client.go:304]: it would've been nice to check if strconv.Itoa returns an error:
newid, err := strconv.Itoa(id)

**Comment by Drynok** [comment]
Good work!

File: client.go (hunk 1 above)
Comment (protsack-stephan) [reply client.go:304]: `strconv.Itoa` does not return error (`Atoi` does but we don't use it in this piece of code). 
https://pkg.go.dev/strconv#Itoa

**Review by sashah2 — APPROVED** [review]
Good work! 💪 

LGTM!

File: client.go (hunk 1 above)
Comment (Drynok) [reply client.go:304]: ok

File: client.go [hunk 2]
```diff
@@ -272,3 +295,70 @@ func (cl *Client) Namespaces(ctx context.Context) ([]Namespace, error) {
...
+		if !user.Missing {
+			users[user.UserID] = user
+		}
+	}
+
+	return users, nil
+}
+
+// Users get single user by id.
+func (cl *Client) User(ctx context.Context, id int) (User, error) {
```
Comment (prabhat393) [inline client.go:352]: Do we need an extra API for this? Can't we just call Users with ids slice containing one element?

**Review by prabhat393 — APPROVED** [review]

File: client.go (hunk 2 above)
Comment (protsack-stephan) [reply client.go:352]: This is done in order to save user of the package from `boilerplate code`. If you decide to call `Users` functions to get users, you need:
a) Check for the error that is returned from the method 
b) Check if `map` that was returned from the method contains that particular id
c) If you want to notify parent function that user was not found you need to create custom error type for that etc.
This makes code that uses `Users`  for single user API call cumbersome and repeatable especially if you want to call in couple handlers. So intention of this piece of code is to solve that problem beforehand by providing convenience function to get single user.

**Comment by prabhat393** [comment]
LGTM.

File: client.go (hunk 2 above)
Comment (prabhat393) [reply client.go:352]: Makes sense

**Comment by Memije** [comment]
LGTM!

**Review by Memije — APPROVED** [review]

**Pull request merged**